cmd/cluster/create.go

```diff
@@ -81,9 +101,14 @@ func createClusterCommand() *cobra.Command {
 				return fmt.Errorf("failed to create arlon app: %s", err)
 			}
 			// Create "cluster app" for cluster
+			if patchRepoUrl != "" {
```

Comment:
Wouldn't it be simpler to test `if overridden {` instead? There are too many variables indicating that overrides are active:
- overridesDir
- patchRepoUrl
- overridden
Pick one to be consistent

Reply:
ok. I'm using overridden everywhere now